### ISSUE TITLE
The ability for anyone to bench another user for a day

### ISSUE THREAD

**Issue by EvanHunzinger** [James-Firth/git-slackin#50]
Being able to bench someone else temporarily for sick days would be very nice.

**Comment by James-Firth** [comment]
Any sort of 'snooze' for notifications and benching seems like it would be a good feature!

Were you thinking _any_ user or just 'manager' types?

**Comment by EvanHunzinger** [comment]
Yeah probably just anyone who manages something, such as the repo or Slack channel.

**Comment by James-Firth** [comment]
Okay I could see `snoozer` or something being a property a user can have that gives them the power to snooze others (which would require a second special flag to set them to non-requestable but temporary)

Then have a cronjob/subprocess of some sort check at X hour every day and un-snooze people who have that special flag set.
